src/redesign/_style.scss

```diff
@@ -528,7 +528,7 @@ select.form-control {
 }
 
 .username-suggestion {
-  padding: 0.15rem 0.5rem;
```

Comment:
This is not the desired change, according to Figma design `padding-left/right` should be 2px and `font-size: 12px` and `line-height: 24px`.

Reply:
<img width="79" alt="Screen Shot 2021-06-24 at 5 52 18 PM" src="https://user-images.githubusercontent.com/15120237/123265879-edc36580-d514-11eb-9fd6-eb31b08df4de.png">
I think this is not promissing with design ?